pages/installation/virtual_machine_appliances.rst

```diff
@@ -60,25 +60,15 @@ credentials.
 
 The web interface is reachable on port 80 at the IP address of your virtual
 machine. The login prompt of the shell is showing you this IP address, too. (See
-screenshot above). DHCP should be enabled in your network otherwise take a look into
-the ``graylog-ctl`` command to apply a static IP address to the appliance.
+screenshot above).
 
-The standard user for the web interface is *admin* with the password *admin*.
+The standard user for the web interface is *admin*, the password is shown on first boot.
```

Comment:
.. in the console of the virtual server.

People tend to not look at that if they do not know they should.